NOTICE.txt

```diff
@@ -5544,6 +5608,40 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+--------------------------------------------------------------------
+Dependency: golang.org/x/sys
```

Comment:
Is this added as dependency of `github.com/alexbrainman/odbc`? If that is the case it should be also removed.

Reply:
I recreated the vendor dir and now I believe that the only dependency added to the NOTICE because of this module is: "github.com/jmoiron/sqlx"

Reply:
Yes, this is looking good now, thanks!